src/Features/CSharp/Portable/CodeLens/CSharpCodeLensDisplayInfoService.cs

```diff
@@ -165,6 +165,34 @@ public string GetDisplayName(SemanticModel semanticModel, SyntaxNode node)
             }
         }
 
+        public string ConstructFullName(SemanticModel semanticModel, SyntaxNode methodNode)
+        {
+            var name = semanticModel.GetDeclaredSymbol(methodNode).Name;
```

Comment:
you have symbol for methodNode. can it just follow up containing symbol to get name? or use ToDisplayString with full namestyle?

calling DeclaredSymbol seems unnecessary.